archive/c/c-sharp/testinfo.yml

```diff
@@ -0,0 +1,9 @@
+folder:
+  extension: ".cs"
+  naming: "pascal"
+
+container:
+  image: "mono"
+  tag: "5.18"  # Cannot use slim because it does not contain mcs for compilation
+  build: "mcs -reference:System.Numerics {{ source.name}}{{ source.extension }}"
```

Comment:
I don't know anything about this, but is it possible to use `csc` to compile and `mono` to run as seen here: https://www.mono-project.com/docs/getting-started/mono-basics/. If so, that may loosen the requirement of having the complete image.

Reply:
For future reference, builds timeout if they take longer than 10 minutes to produce log output. Also, public repos have a 50 minute limit on builds. We may run into some limit eventually that we'll have to try to circumvent.

Reply:
Mono is the c# compiler for mono. csc is the Roslyn compiler. They both now exist within mono, but we need the full image in order to use either. The slim image doesn't contain any sdk at all.

Reply:
Haha fair enough! This is a tough one.

Reply:
I think, I'll just need to find a way to do log output while it's downloading the images.

Reply:
Sounds like a plan. Too bad there isn't a nice workaround. It's fine locally, but downloading and building it every time in Travis is going to be time consuming.